pages/index.tsx

```diff
@@ -4,11 +4,11 @@ import { Button } from '~/components/Button';
 import { TitleText } from '~/components/TitleText';
 import { Input } from '../components/Input';
 import { DeleteModal } from '~/components/DeleteModal';
-import jsonServerFunctions from '~/utils/jsonServerFunctions';
+import { createGift, getAllGifts } from '~/utils/jsonServerFunctions';
 
 const inter = Inter({ subsets: ['latin'] });
 
-export type FullLocalStorage = {
+export type Gifts = {
```

Comment:
Tyypin nimi olisi parempi olla yksikössä, kun kyseessä on vain yksi lahja
```suggestion
export type Gift = {
```